github-to-zulip/pull-reminders.sh

```diff
@@ -5,10 +5,19 @@ set -euo pipefail
 # rm -rf /tmp/akvo/github-pull-reminders
 mkdir -p /tmp/akvo/github-pull-reminders
 
+github_fetch() {
+    HEADERS=()
+    HEADERS+=("-HAccept: application/vnd.github.v3+json")
+    if [ -n "${GITHUB_TOKEN}" ];
```

Comment:
Since we're only bash, I suggest to *always* use `[[` instead of `[`

https://unix.stackexchange.com/a/306115

Reply:
Thanks, I'll change this.